Formula/gitui.rb

```diff
@@ -40,8 +41,8 @@ def install
     # remove ANSI colors
     screenlog.gsub! /\e\[([;\d]+)?m/, ""
     assert_match "Author: Stephan Dilly", screenlog
-    assert_match "Date: 2020-06-15", screenlog
-    assert_match "SHA: 9c2a31846c417d8775a346ceaf38e77b710d3aab", screenlog
+    assert_match "Date: 2020-07-06", screenlog
+    assert_match "SHA: 2c53c0948788adc732fe5273f0288065224cac4d", screenlog
```

Comment:
You should either always check the same tag, or don't check the hash/date. This is really difficult to keep up to date.

Reply:
@SMillerDev  see above